docs/snippets/install-kn.md

```diff
@@ -20,15 +20,13 @@ The `kn` CLI also simplifies completion of otherwise complex procedures such as
         brew install knative/client/kn
         ```
 
-        ??? bug "Having issues upgrading `kn` using Homebrew?"
+        If you are having issues upgrading using Homebrew, it might be due to a change to a CLI repository where the `master` branch was renamed to `main`. Resolve this issue by running the command:
```

Comment:
We should look at when this comment was added.  If it's more than 2 years old, we can probably retire it.